### PR TITLE
Connection between auth-service and active-sessions, small changes to browser-facade

### PR DESCRIPTION
**Pull request by jakublaz**
data validation done in browser-facade, connection  to get session id between auth-service and active-sessions is established

### CODE REVIEW

File: docker-compose.yml
Comment (piotr-ladkowski) [inline docker-compose.yml:1]: it is not a good idea to use auth network here but rather use a new network

File: auth-service/build.rs
Comment (piotr-ladkowski) [inline auth-service/build.rs:1]: it is not a good idea to modify this - will probably lead to conflicts with Wiktor working on his branch.

File: active-sessions/src/main.rs
Comment (piotr-ladkowski) [inline active-sessions/src/main.rs:1]: same as in auth-service/build.rs

File: active-sessions/src/client.rs
Comment (piotr-ladkowski) [inline active-sessions/src/client.rs:1]: see auth-service/build.rs

File: active-sessions/Cargo.toml
Comment (piotr-ladkowski) [inline active-sessions/Cargo.toml:1]: same as in auth-service/build.rs

File: active-sessions/Dockerfile
Comment (piotr-ladkowski) [inline active-sessions/Dockerfile:1]: auth-service/build.rs

File: active-sessions/proto/active-sessions.proto
Comment (piotr-ladkowski) [inline active-sessions/proto/active-sessions.proto:1]: auth-service/build.rs

**Review by piotr-ladkowski — CHANGES_REQUESTED** [review]

File: auth-service/build.rs
Comment (jakublaz) [reply auth-service/build.rs:1]: this is my service and wiktor is not working on this service i have made sure of it

File: docker-compose.yml
Comment (jakublaz) [reply docker-compose.yml:1]: ok we can do it, but to be consistient we should do it will all the databases. OK?

**Comment by jakublaz** [comment]
I have made sure that wiktor is not working on this branch he is doing docs now for message-data-centre

**Review by piotr-ladkowski — APPROVED** [review]

**Pull request merged**